config/default/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 # Adds namespace to all resources.
-namespace: performance-addon-operators-system
+#namespace: performance-addon-operators-system
```

Comment:
why comment out and not delete?

Reply:
probably a forgotten wip comment,  will delete, thanks.

Reply:
Actually I take that back. This is like a template. At some point we may want this feature, but we wouldn't know about the existence. Too much information around.

The same as in:
# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
# crd/kustomization.yaml
#- manager_webhook_patch.yaml

That is a good way to know how to add stuff.
I'll keep it for now and let's take it to the meeting.